build-config/scripts/onie-mk-itb.sh

```diff
@@ -139,6 +141,7 @@ DTB="$(realpath $DTB)"
 			arch = "$ARCH";
 			os = "linux";
 			compression = "none";
+			load = <$FDT_LOAD>;
```

Comment:
Looking at machine `nxp_p2020rdbpca-r0` based on u-boot version u-boot-2013.01.01.  When the dtb load address is not specified, u-boot just uses the dtb data from where ever it found it inside the .itb file.

See the call to `fit_image_get_load()` around line 1627 in u-boot-2013.01.01/common/image.c

Also look at this boot output from a `nxp_p2020rdbpca-r0`:

```
## Flattened Device Tree from FIT Image at 01000000
   Using 'nxp_p2020rdbpca' configuration
   Trying 'dtb' FDT blob subimage
     Description:  nxp_p2020rdbpca-r0.dtb
     Type:         Flat Device Tree
     Compression:  uncompressed
     Data Start:   0x0156effc
     Data Size:    12629 Bytes = 12.3 KiB
     Architecture: PowerPC
     Hash algo:    crc32
     Hash value:   62511f4b
   Verifying Hash Integrity ... crc32+ OK
   Booting using the fdt blob at 0x0156effc
```

Notice how `Data Start` and `Booting using the fdt blob at` both specify `0x0156effc`.  If the dtb section had had a `load` address the addresses would have been different.

So this is the first machine that needs this.  My first thought is "does this machine really need to specify the load address of the dtb?"  Does it work without specifying the dtb `load` address?  That would be the simplest solution.  Do that experiment first.

If the dtb `load` address is really needed we will think of something.